packages/optimizer/README.md

```diff
@@ -108,6 +108,29 @@ Automatically import any missing AMP Extensions (e.g. amp-carousel).
 - default: `true`
 - used by: [AutoExtensionImport](lib/transformers/AddMandatoryTags.js)
 
+#### `fetch`
+
+Provide a custom fetch handler. You can use this option to configure a custom proxy server. Example:
+
+```js
+const nodeFetch = require('node-fetch');
+
+const proxyHost = '...';
+const proxyPort = '...';
+
+const fetch = (url, opts={}) => {
+  opts.agent = new HttpsProxyAgent(`${proxyHost}:${proxyPort}');
+  return nodeFetch(url, opts)
+}
+const optimizer = AmpOptimizer.create({
+  fetch,
+});
+```
+
+- name: `fetch`
+- valid options: a [whatwg fetch](https://github.com/whatwg/fetch) compatible fetch implementation. 
+- default: `[node-fetch](https://www.npmjs.com/package/node-fetch)`
```

Comment:
did you mean to wrap a markdown link inside of code ticks?

![image](https://user-images.githubusercontent.com/465414/84332396-d4928580-ab5a-11ea-888c-fdbba2078677.png)

Reply:
yes, this was intenational, but not sure if it makes sense ;-)